flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/FlinkService.java

```diff
@@ -653,7 +655,10 @@ public SavepointFetchResult fetchSavepointInfo(
     public void disposeSavepoint(String savepointPath, Configuration conf) throws Exception {
         try (RestClusterClient<String> clusterClient =
                 (RestClusterClient<String>) getClusterClient(conf)) {
-            clusterClient.disposeSavepoint(savepointPath);
+            clusterClient.sendRequest(
```

Comment:
I check the `disposeSavepoint` internal implementation, why current way is better?

Reply:
The disposeSavepoint method of the RestClusterClient, first triggers the dispose action and then keeps polling Flink until it completes.

The way we use it, this is completely unnecessary as we do not handle/retry dispose actions from our side anyways. This way we can save some rest api calls.

However we lose the error logs unfortunately if the dispose actually failed.

Reply:
Got it.